manuscripts/method/03.results.md

```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
 
-### DDMC correctly predicts AKT1 and ERK2 as upstream kinases of signaling clusters
+<!-- Model behavior -->
+Next, we explored how using different weights affects the overall phosphorylation signal and sequence information of the resulting clusters. To do so, we compared the model behavior after clustering the CPTAC data with a weight of 0 (data only), 20 (mix), and 50 (mainly sequence). First, we hypothesized that the data model would generate clusters with less within-cluster phosphorylation signal variation and thus a lower mean squared error (MSE). To test this, we computed the average peptide-to-cluster MSE of 2000 randomly selected peptides for each model. Although the differences were not significant, we did observe an direct correlation between weight and MSE (Figure {@fig:preds} B). Next, we calculated the cumulative PSSM enrichment by summing the information (bits) of all cluster PSSMs per model. Importantly, we observe a substantial increase in cumulative cluster information as the weight increases (Figure {@fig:preds} C). To further illustrate the clustering behavior, we tracked TBC1D5 S584-p in the three models and observed that in addition to the obvious increase in overall motif enrichment, as the weight increases the cluster PSSM is progressively more representative of this peptide (Figures {@fig:preds} D-H). These results show that using a mixed weight to prioritize both information sources—data and sequence—leads to clusters of phosphosites with similar phosphorylation behavior and sequence motifs. 
 
-![**Validation of upstream kinase predictions.** (A-B) PCA analysis of the phosphorylation levels of MCF7 cells exposed to a drug screening. C) Upstream kinase prediction of EBDT's AKT1 and CPTAC's AKT1 cluster. D) Heatmap showing the effect of inhibitors to the phosphorylation signal of cluster 1. E) PhosphoSitePlus-reported AKT1 substrates per cluster (F) Resulting PSSM generated using ERK2 substrates reported by Carlson et al [@PMID:PMC3779841]. (G) Upstream kinase predictions of CPTAC clusters 7, 9, 13, and 21 in addition to the ERK2 motif shown in (F). H) Upstream kinase predcitions of the same PSSMs after randomly shuffling the motif positions.](figureM3.svg "Figure 3"){#fig:TvsNAT width="100%"}
+<!-- Notes figureM3:
+1. Statistical test in C?
+2. G does not really resemble TBC1D5's peptide sequence... Maybe it'd be worth finding a more obvious example. Or even remove D-H? --> 
```

Comment:
What's special about this particular peptide? I like everything else but this in particular seems weird about (D).

Reply:
It's just an example of a peptide that shows the general trends described in the first three plots. What do you find weird about (D)? @aarmey

Reply:
I think singling out any single peptide will beg the question why you picked that one.

Reply:
The rationale for singling out a peptide is to help conceptualize the point of A-C. It shows that it follows the general trends and then plotting the motifs for each model illustrates that increasing the weight makes the cluster motif be more representative of that peptide sequence.